contracts/extensions/HypERC721URIStorage.sol

```diff
@@ -64,7 +64,7 @@ contract HypERC721URIStorage is HypERC721, ERC721URIStorageUpgradeable {
     function supportsInterface(bytes4 interfaceId)
         public
         view
-        override(ERC721EnumerableUpgradeable, ERC721Upgradeable)
+        override(ERC721EnumerableUpgradeable, ERC721URIStorageUpgradeable)
```

Comment:
Seems unrelated?

Reply:
I am not able to compile without this change. Following is the error I am getting :- 

TypeError: Function needs to specify overridden contract "ERC721URIStorageUpgradeable".
  --> contracts/extensions/HypERC721URIStorage.sol:67:9:
   |
67 |         override(ERC721EnumerableUpgradeable, ERC721Upgradeable)
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Note: This contract: 
  --> @openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721URIStorageUpgradeable.sol:13:1:
   |
13 | abstract contract ERC721URIStorageUpgradeable is Initializable, IERC4906Upgradeable, ERC721Upgradeable {
   | ^ (Relevant source part starts here and spans across multiple lines).


TypeError: Invalid contract specified in override list: "ERC721Upgradeable".
  --> contracts/extensions/HypERC721URIStorage.sol:67:9:
   |
67 |         override(ERC721EnumerableUpgradeable, ERC721Upgradeable)
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Note: This contract: 
  --> @openzeppelin/contracts-upgradeable/token/ERC721/ERC721Upgradeable.sol:20:1:
   |
20 | contract ERC721Upgradeable is Initializable, ContextUpgradeable, ERC165Upgradeable, IERC721Upgradeable, IERC721MetadataUpgradeable {
   | ^ (Relevant source part starts here and spans across multiple lines).


Error HH600: Compilation failed

Reply:
Ah yes i see